frontend/express/public/javascripts/countly/vue/templates/datatable.html

```diff
@@ -74,8 +74,14 @@
                                     :options.sync="getMatching(availableDynamicCols)">
                                     </cly-checklistbox>
                                 </div>
-                                <slot name="export-config"></slot>
                             </div>
+                            <div v-if="customExportFileName" class="cly-vue-eldatatable__export--fileExport">
+                            <div  class="bu-my-4 bu-mx-4">
+                                <span class="text-medium"> {{i18n('export.file-name')}} </span>
```

Comment:
Can you match this to the design, The margin below filename does not seem accurate
Also, the indentation for html is incorrect